tests/cms/aod/test_local_dataset.py

```diff
@@ -1,4 +1,4 @@
-from .config import f_location
+from tests.cms.aod.config import f_location
```

Comment:
Why did this have to change - shouldn't it have just worked?

Reply:
Import from .config works when running the long test. But if I want to `pytest test_local_dataset.py` alone when my path is in `tests/cms/aod,` it could not find the correct file...